### DISCUSSION TITLE
Feature Request - project boards on mobile

### DISCUSSION THREAD

**Discussion by jeffrey-tc** [community/community#3653] [Mobile]
It would be nice to be able to view and edit project boards on mobile. We use project boards on a lot of different repositories and organizations and sometimes you want to quickly manage issues on a project board while away from a computer.

**Comment by aconsuegra** [comment]
Thanks for your feedback @jtaylorchang!

We are already exploring the bets way to display and interact with Project board in the mobile apps. Stay tuned!
